### PR TITLE
🔧 transfer assetHub

### PR DESCRIPTION
**Pull request by roiLeo**
- ref #7632 

What need to be tested:
- transfer on ahk
- transfer on ahp
- transfer on old ahp collection (`u-`)

### CODE REVIEW

File: composables/transaction/transactionSend.ts [hunk 1]
```diff
@@ -69,12 +66,13 @@ function execSendBasilisk(item: ActionSend, api, executeTransaction) {
...
 // https://github.com/paritytech/substrate/blob/e6a13b807a88d25aa1cd0d320edb9412c3692c67/frame/uniques/src/functions.rs#LL58C2-L58C51
-function execSendStatemine(item: ActionSend, api, executeTransaction) {
+function execSendAssetHub(item: ActionSend, api, executeTransaction) {
   const legacy = isLegacy(item.nftId)
   const paramResolver = assetHubParamResolver(legacy)
 
+  // getApiCall(api, item.urlPrefix, Interaction.SEND, legacy)
   executeTransaction({
-    cb: getApiCall(api, item.urlPrefix, Interaction.SEND),
+    cb: api.tx.uniques.transfer,
```
Comment (vikiival) [inline composables/transaction/transactionSend.ts:75]:
```suggestion
    cb: getApiCall(api, item.urlPrefix, Interaction.SEND, legacy),
```

**Review by vikiival — CHANGES_REQUESTED** [review]

File: composables/transaction/transactionSend.ts (hunk 1 above)
Comment (roiLeo) [reply composables/transaction/transactionSend.ts]: this will be the same issue imo

File: composables/transaction/transactionSend.ts [hunk 2]
```diff
@@ -1,7 +1,4 @@
-import {
-  assetHubParamResolver,
-  getApiCall,
-} from '@/utils/gallery/abstractCalls'
+import { assetHubParamResolver } from '@/utils/gallery/abstractCalls'
```
Comment (vikiival) [inline composables/transaction/transactionSend.ts:1]:
```suggestion
import { assetHubParamResolver, getApiCall } from '@/utils/gallery/abstractCalls'
```

**Comment by vikiival** [comment]
well accourding to screen that was in the issue the point is that the transaction is constructed correctly but instead of `uniques.transfer`, `nfts.transfer` is sent

**Comment by roiLeo** [comment]
> well accourding to screen that was in the issue the point is that the transaction is constructed correctly but instead of `uniques.transfer`, `nfts.transfer` is sent

legacy collection (prefix `u-`) should use `uniques` or `nft` pallet?

File: composables/transaction/transactionSend.ts [hunk 3]
```diff
@@ -1,7 +1,4 @@
-import {
-  assetHubParamResolver,
-  getApiCall,
-} from '@/utils/gallery/abstractCalls'
+import { assetHubParamResolver, getApiCall } from '@/utils/gallery/abstractCalls'
```
Comment (roiLeo) [inline composables/transaction/transactionSend.ts:1]:
```suggestion
import {
  assetHubParamResolver,
  getApiCall,
} from '@/utils/gallery/abstractCalls'
```

**Comment by prury** [comment]
@roiLeo when you think is ready lmk, i'll ask the user to test using the deploy link

**Comment by vikiival** [comment]
> legacy collection (prefix u-) should use uniques or nft pallet?

Uniques.

**Comment by roiLeo** [comment]
> @roiLeo when you think is ready lmk, i'll ask the user to test using the deploy link

this is already testable

**Comment by vikiival** [comment]
Screenshot 2023-10-16 at 12 53 56


Screenshot 2023-10-16 at 12 53 59

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Tested, the correct handler is spawned

**Comment by roiLeo** [comment]
oh my.. this screen resolution should be illegal

**Pull request merged**